main.go

```diff
@@ -72,8 +87,56 @@ func getRepos(dir, user, maxrepo string, limit []string) (repos []string) {
 
 		// Get first column from 'gh repo list' output, it's repo name
 		words := strings.Split(strs[i], "\t")
-		repo := words[0]
+		repos = append(repos, words[0])
+	}
+
+	// Clone repos
+	cloneRepos(repos, limit, dir)
+
+	return
+}
+
+// getStars get list of starred reopsitories and clone it
+func getStars(dir, user, maxrepo string, limit []string) (repos []string) {
+
+	// Get stars by github api
+	resp, err := http.Get(fmt.Sprintf("https://api.github.com/users/%s/starred?page=1&per_page=%s", user, maxrepo))
```

Comment:
This don't work like gh client. per_page value must be 100 or lower https://docs.github.com/en/rest/activity/starring#list-repositories-starred-by-a-user
Look at https://github.com/Kirill/github-stars-backup/blob/b4f30e9ec6a90ebea3ff1a0aabef62519381869d/main.go#L62
it's not best solution. For example, I can't clone one repo - I have it starred, but for this moment I don't see it (user in block-list, or deleted early I don't know). If repo can't be cloned this don't stop process.

Reply:
May be use request without parameters: 
/users/{username}/starred

Reply:
WELL!

I rewrote the getStars function with gh utility.
Now I can see your repositories with over 500 stars.
I have added the -printonly application flag, it can be used for testing, it shows all selected repositories but does not clone them.

Please review again.
I also made a lot of small changes, it's not that important.